docs/uplo/downloads/v1.md

```diff
@@ -0,0 +1,79 @@
+<style>
+@media (min-width: 980px) {
+    .md-nav, .md-sidebar {
+      display: none!important;
+    }
+}
+</style>
+
+# Uplo Downloads Measure
+
+<div id="value-display"></div>
+<strong>Description:</strong>
+<div class="tile-1" style="text-align:center; color:black">
+Total number of times a work has been downloaded, as determined by processing the Access Logs. To count multiple downloads of the same file by the same user, there must be a gap of 30 minutes from the last counted download.
```

Comment:
There should probably mention that the UPLO measures are specific to the UPLO website. Otherwise people might wonder what the different between metrics.operas-eu.org/up-logs/downloads/v1 and metrics.operas-eu.org/uplo/downloads/v1 if they have identical definitions?

This also applies to uplo views and uplo reads